platforms/bukkit/common/src/main/java/com/dfsek/terra/bukkit/util/VersionUtil.java

```diff
@@ -77,13 +77,13 @@ public boolean isSpigot() {
     public static final class MinecraftVersionInfo {
         private static final Logger logger = LoggerFactory.getLogger(MinecraftVersionInfo.class);
 
-        private static final Pattern VERSION_PATTERN = Pattern.compile("v?(\\d+)_(\\d+)_R(\\d+)");
+        private static final Pattern VERSION_PATTERN = Pattern.compile("(\\d+)\\.(\\d+)\\.(\\d+)");
         private final int major;
         private final int minor;
         private final int patch;
 
         private MinecraftVersionInfo() {
-            this(Bukkit.getServer().getClass().getPackage().getName().split("\\.")[3]);
+            this(Bukkit.getServer().getMinecraftVersion());
```

Comment:
This is a Paper-only method. According to https://terra.polydev.org/install/bukkit.html Spigot is still supported (albeit not recommended).

Perhaps you could do something with `Bukkit.getServer().getBukkitVersion().split("-")[0]`

Reply:
Ahh I wasn't aware that Spigot was still supported, I'll make that change